eplaunch/workflows/default/file_details.py

```diff
@@ -11,10 +12,10 @@ class ColumnNames:
 class FileDetailsWorkflow1(BaseEPLaunchWorkflow1):
 
     def name(self):
-        return "Get File Details 1.0"
+        return "Get File Details"
 
     def context(self):
-        return "EPLaunch 2.9.2"
+        return f"{NAME} {VERSION}"
```

Comment:
Dynamically retrieving the current name and version so that we don't have this hardcoded anymore.